layouts/index.html

```diff
@@ -0,0 +1,157 @@
+{{ define "body" }}
+<div class="container-fluid">
```

Comment:
I really like what you've done with the design here - could we make the width of the content match the width of the `<hr />`s though? Maybe setting this to have `style="width: 80%"` and then resetting the `<hr />`s to `class="w-100 mx-auto"` would be a bit more consistent IMO!

Reply:
I feel like `style="width: 80%"` creates too much whitespace on the sides.

Reply:
That's fair. It just initially felt off to me that the title text + the `<hr />`s and the content below it are all at different widths, but I'll let you make the call.

![image](https://user-images.githubusercontent.com/45278276/187467588-1d7f5da3-9455-44f2-8419-e0affd902896.png)